.gitlab-ci.yml

```diff
@@ -0,0 +1,24 @@
+image: php:7.1-cli-alpine
```

Comment:
Why using PHP 7.1?

Reply:
This is just an import: will likely restrict this to 7.3+, or 7.2 depending on the downstream project that consumes this.